packages/react/src/generators/library/lib/add-rollup-build-target.ts

```diff
@@ -30,6 +30,7 @@ export async function addRollupBuildTarget(
     host,
     {},
     {
+      '@nx/rollup': nxVersion,
```

Comment:
The weird thing is that on line 23 of the same file we're doing `ensurePackage` for the `@nx/rollup` package, and it still doesn't get installed :/

Reply:
it doesn't get added to the workspace, i think it needs to be in the init generator

Reply:
We need to fix this in `@nx/rollup:init` generator. The bug is that `@nx/rollup` package is only added when compiler is swc. It should be added for all cases.